recipe/meta.yaml

```diff
@@ -1,29 +1,26 @@
-{% set name = "pangaea" %}
 {% set version = "0.0.4" %}
-{% set sha256 = "a114424e34e42befa691ddfe5f07272867f095fdc4f28dca633d5c6cadf26c16" %}
 
 package:
-  name: {{ name|lower }}
+  name: pangaea
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/p/pangaea/pangaea-{{ version }}.tar.gz
+  sha256: a114424e34e42befa691ddfe5f07272867f095fdc4f28dca633d5c6cadf26c16
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
     - python
-    - setuptools
+    - pip
   run:
     - python
     - gazar
     - netcdf4
-    - pynio     # [not py3k and not win]
+    - pynio  # [not py3k and not win]
```

Comment:
@snowman2 is this an optional dependency?  If we can remove that we can make this recipe `noarch` and build it much faster.

Reply:
According to the `setup.py` upstream only  `gazar` and `wrf-python` are mandatory dependencies.

Reply:
is noarch fully supported now? i.e. should I be using that for pure python recipes?

Reply:
If I remember correctly, that should be fine.

Reply:
> is noarch fully supported now?

The only issue with `noarch` is when using the package with `conda-constructor`. If you are not using constructor and `pynio` dependency can be removed I highly recommend making this `noarch`.